susecloud

```diff
@@ -285,6 +285,16 @@ else
 fi
 
 #############################################################
+section_header "Ceph report"
+
+if [ -e /usr/bin/ceph ]; then
+    ceph_report_dir=$(mktemp -d)
```

Comment:
I know @aspiers wasn't keen on using /var/log in https://github.com/SUSE-Cloud/supportutils-plugin-susecloud/pull/33 but I actually feel it's the best place to put this, as `split-supportconfig` will then extract the log in the right place.

Or should we create a /var/log/supportconfig directory for this kind of "temporary" files?